AutoTenez.py

```diff
@@ -214,170 +51,336 @@ def make_reservation(headers, cookies, date_tomorrow, md5slotkey, your_external_
+            + str(self.reservation_date) + "&md5slotkey=" + md5slotkey + "&externalReferences=" + self.your_external_reference + "," + self.other_players_external_references, \
+            headers=self.headers, cookies=self.cookies)
+        
+        # We expect a status code 204 No Content, with of course no content
+        if (r.status_code == 204) and (len(r.content) == 0):
+            print("Successfully made the reservation! With md5slotkey " + md5slotkey)
+        else:
+            print("Failed to make the reservation with md5slotkey " + md5slotkey + ". Server returned status code: " + r.status_code())
+
+# If ran from command line
+if __name__ == "__main__":
+    try:
+        # Parse input arguments
+        parser = argparse.ArgumentParser(description='Reserve tennis court for tomorrow.')
+        parser.add_argument('-c',  '--courts',               nargs='+',     help='Specify courts ("Baan X", where X is the court number). Default setting is all courts.')
+        parser.add_argument('-d',  '--date',                                help='Specify the date to make the reservation (yyyy-mm-dd). Defaults to `reservation_date`.')
+        parser.add_argument('-t2', '--time_second_choice',   nargs='+',     help='Time you would like to reserve for the second option (hh:mm).')
+        parser.add_argument('-c2', '--courts_second_choice', nargs='+',     help='Specify courts for the second option.')
+        parser.add_argument(       '--dryrun',               default=False, help="Pass as an argument to only check available time slots, but don't make a reservation.")
+
+        required_arguments = parser.add_argument_group('Required arguments')
+        required_arguments.add_argument('-t', '--time',    nargs='+', help="Time you would to you reserve. One or two consecutive times are allowed (hh:mm).", required=True)
+        required_arguments.add_argument('-f', '--friends', nargs='+', help="With whom you would like to play. 1-3 additional players allowed.", required=True)
+
+        args = parser.parse_args()
+
+        # Process input arguments
+        player2_external_reference = args.friends[0]
+        player3_external_reference = ""
+        player4_external_reference = ""
+        if len(args.friends) > 1:
+            player3_external_reference = args.friends[1]
+        if len(args.friends) > 2:
+            player4_external_reference = args.friends[2]
+
+        reservation_date = ""
+        if (args.date):
+            reservation_date = args.date
+
+        first_choice_first_hour = args.time[0]
+        first_choice_second_hour = ""
+        if len(args.time) > 1:
+            first_choice_second_hour = args.time[1]
+
+        first_choice_courts = []
+        if args.courts:
+            first_choice_courts = args.courts
+
+        second_choice_first_hour = ""
+        second_choice_second_hour = ""
```

Comment:
```suggestion
        second_choice_first_hour = None
        second_choice_second_hour = None
```
- Same initial value as the default value in `find_time_slot(self, slots, first_hour, second_hour=None, courts=[]):`